packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -52,7 +57,7 @@ export default class MeetingTeamPrompt extends Meeting {
       phases,
       facilitatorUserId,
       meetingType: 'teamPrompt',
-      name: name ?? createTeamPromptDefaultTitle(),
+      name: createTeamPromptTitle(name || 'Standup', new Date(), 'UTC'),
```

Comment:
+1 Is it possible to use the user's time zone here? It would be weird for someone in Australia to start a meeting in the morning and have it show as the previous day.

Reply:
yeah well, this issue existed before this change anyway as we were using server time so there's no regression here. to pass user timezone we'd need to send it from the client (preferably via http header) and pass it around via gql context. I wanted to avoid such a big scope creep in this PR as it's a major change but I'll create a task so we can outline the solution and fix it for good

Reply:
Gotcha - maybe add a TODO here so we know this is something we should improve eventually?

Reply:
yeah, good idea